docusaurus/docs/creating-a-plugin.mdx

```diff
@@ -47,26 +47,26 @@ Give your plugin a description. This will help users more easily understand what
 
 Select from the following choices:
 
-- **app** (Applications, or app plugins, create a custom out-of-the-box monitoring experience.)
-- **datasource** (Data source plugins add support for new databases or external APIs.)
-- **panel** (Add new visualizations to dashboards with panel plugins.)
+- **app**: Create a custom out-of-the-box monitoring experience.
+- **datasource**: Add support for new databases or external APIs.
+- **panel**: Add new visualizations to dashboards.
 
 To learn more about the types of plugins, refer to the [plugin management guidelines](https://grafana.com/docs/grafana/latest/administration/plugin-management/).
 
 ### Do you want a backend part of your plugin?
 
-App and Datasource plugins can have a backend component written in Go. Developing a backend to your plugin brings powerful features such as:
+App and datasource plugins can have a backend component written in Go. Backend plugins offer powerful features such as:
```

Comment:
Normally, data source is two words. However, sometimes in the UI it's one word or there are a lot of uses of "datasource" instead of "data souce", making clean-up out of scope. 
```suggestion
App and data source plugins can have a backend component written in Go. Backend plugins offer powerful features such as:
```